files/config/proxy/graphdb.properties

```diff
@@ -0,0 +1,462 @@
+#
+# graphdb.connector.truststoreFile = <path-to-custom-truststore-file>
+# graphdb.connector.truststorePass = <secret>
+#
+##### X.509 CERTIFICATE AUTHENTICATION REVOCATION STATUS CHECK CONFIGURATION ######
+#
+# Controls whether OCSP checks are performed, true by default.
+# graphdb.auth.methods.x509.ocsp = true
+#
+# Controls whether CRLDP checks are performed, true by default.
+# graphdb.auth.methods.x509.crldp = true
+#
+# Uncomment the following to set a Certificate Revocation List to Tomcat,
+# which will allow revocation checks for certificates that do not provide
+# an Authority Information Access (AIA) extension, or as an alternative
+# in case of OCSP or CrlDP responders downtime.
+# graphdb.auth.methods.x509.crlFile = <path-to-certificate-revocation-list>
+#
+
+###### AUDIT TRAIL ######
+#
+# Enable the detail audit trail for all operations with the minimal access role of:
+# USER, REPO_MANAGER, ADMIN, ANY
+#
+# graphdb.audit.role = USER
+#
+# and read, write operations with the minimal access of:
+# READ, WRITE
+#
+# graphdb.audit.repository = READ
+#
+# Audit the specific headers
+#
+# graphdb.audit.headers = Origin, Host
+#
+# List headers that should be included into logs in front of client requests.
+#
+# graphdb.request.id.alternatives = X-Amz-Request-Id
+#
+# Enable logging of "X-Request-Id" header and listed in graphdb.request.id.alternatives headers into any log.
+# "X-Request-Id" header will be included by default. In case "X-Request-Id" is not present in the headers of
+# the request it will be generated randomly in UUID type 5 format.
+#
+# graphdb.append.request.id.headers = true
+
+###### CLUSTER CONFIGURATIONS ######
+
+# Embedded cluster proxy configurations
+
+# Uncomment to disable the embedded cluster proxy that redirects requests to the leader in cluster deployment.
+# Will be disabled automatically if external proxy is used.
+#graphdb.cluster.proxy.enabled=false
+
+# The following configurations represents the defaults of the HTTP client used to redirect requests.
+# Uncomment and change any of the default values
+# -1 means infinite timeout
+# The connection timeout is in seconds
+#graphdb.cluster.proxy.socketTimeout=-1
+#graphdb.cluster.proxy.connectionTimeoutS=15
+# This limits the maximum number of concurrent requests send to the leader
+#graphdb.cluster.proxy.maxConnectionsPerRoute=30000
+#graphdb.cluster.proxy.maxConnectionsTotal=50000
+
+# Dedicated cluster proxy configurations
+
+# List the addresses of GraphDB HTTP or RPC address to the nodes that are part of a cluster
+# Note that all of the addresses need to be from the same cluster
+graphdb.proxy.hosts={{- range $i, $node_index := until ( (int $.Values.graphdb.clusterConfig.nodesCount) )}}http://graphdb-node-{{ $node_index }}.graphdb-node.{{ $.Release.Namespace }}.svc.cluster.local:7200{{- if gt (sub (int $.Values.graphdb.clusterConfig.nodesCount) 1 ) $node_index }},{{- end }}
```

Comment:
TBH, this configuration is very concealed being here surrounded by all of those commented properties.